src/service/programService.js

```diff
@@ -539,23 +539,25 @@ function getNominationsList(req, response) {
 
         const orgMap = _.map(orgList, org => {
           return getOrgDetails(req, org);
-        })
+        });
 
-        forkJoin(...userMap, ...orgMap).subscribe(([resData, orgData]) => {
+        forkJoin(...userMap, ...orgMap).subscribe((resData) => {
 
-          _.forEach(resData, function (data, index) {
-            if (data.data.result) {
-              result[index].dataValues.userData = data.data.result.User[0];
+          _.forEach(resData, function (data) {
+            if (data.data.result && data.data.result.User) {
+              const userData = data.data.result.User[0];
+              const index = _.indexOf(_.map(result, 'user_id'), userData.userId)
+              result[index].dataValues.userData = userData;
             }
-          });
 
-          if (!_.isEmpty(orgList)) {
-            _.forEach(orgData, function (data, index) {
-              if (data.data.result) {
-                result[index].dataValues.orgData = data.data.result.Org[0];
+            if (data.data.result && data.data.result.Org) {
```

Comment:
can this be an empty array `data.data.result.Org` ? If yes please do check for isEmpty.